.devcontainer/features/review-scanners/install.sh

```
@@ -0,0 +1,156 @@
+source "${FEATURE_DIR}/../languages/shared/feature-utils.sh" 2>/dev/null || {
+    RED='\033[0;31m'; GREEN='\033[0;32m'; YELLOW='\033[1;33m'; BLUE='\033[0;34m'; NC='\033[0m'
+    log_info() { echo -e "${BLUE}[INFO]${NC} $*"; }
+    log_success() { echo -e "${GREEN}[SUCCESS]${NC} $*"; }
+    log_warning() { echo -e "${YELLOW}[WARNING]${NC} $*"; }
+    log_error() { echo -e "${RED}[ERROR]${NC} $*" >&2; }
+    ok() { echo -e "${GREEN}✓${NC} $*"; }
+    warn() { echo -e "${YELLOW}⚠${NC} $*"; }
+    detect_arch() { case "$(uname -m)" in x86_64) ARCH_UNAME=x86_64; ARCH_DEB=amd64;; aarch64|arm64) ARCH_UNAME=aarch64; ARCH_DEB=arm64;; esac; export ARCH_UNAME ARCH_DEB; }
+    get_github_latest_version() { curl -fsS "https://api.github.com/repos/$1/releases/latest" 2>/dev/null | sed -n 's/.*"tag_name": *"v\?\([^"]*\)".*/\1/p' | head -n1; }
+    get_github_latest_version_or_empty() { get_github_latest_version "$1" 2>/dev/null || echo ""; }
+    apt_update_once() { sudo apt-get update; }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)